src/main/sensors/battery.c

```diff
@@ -92,7 +92,7 @@ void updateBattery(void)
     uint16_t vbatMeasured = batteryAdcToVoltage(vbatLatestADC);
 
     /* battery has just been connected*/
-    if (batteryState == BATTERY_NOT_PRESENT && (ARMING_FLAG(ARMED) || (vbat > batteryConfig->batterynotpresentlevel && ABS(vbatMeasured - batteryAdcToVoltage(vbatPreviousADC)) <= VBAT_STABLE_MAX_DELTA))) {
+    if (batteryState == BATTERY_NOT_PRESENT && vbat > batteryConfig->batterynotpresentlevel && ABS(vbatMeasured - batteryAdcToVoltage(vbatPreviousADC)) <= VBAT_STABLE_MAX_DELTA) {
```

Comment:
Why are you changing this? The idea here is to make sure that we are not in BATTERY_NOT_PRESENT after arming, and i think this is how it should be.